sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt

```diff
@@ -22,27 +22,40 @@ import com.datadog.android.plugin.Feature
 import com.datadog.android.rum.GlobalRum
 import com.datadog.android.rum.RumMonitor
 import com.datadog.android.rum.tracking.NavigationViewTrackingStrategy
+import com.datadog.android.sample.data.RemoteDataSource
 import com.datadog.android.sample.picture.PictureViewModel
 import com.datadog.android.sample.user.UserFragment
 import com.datadog.android.timber.DatadogTree
 import com.datadog.android.tracing.AndroidTracer
 import com.datadog.android.tracing.TracingInterceptor
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
 import com.google.gson.JsonArray
 import com.google.gson.JsonObject
 import io.opentracing.util.GlobalTracer
 import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
 import timber.log.Timber
 
 class SampleApplication : Application() {
 
-    private val tracedHosts = listOf("datadoghq.com", "127.0.0.1")
+    private val tracedHosts = listOf("datadoghq.com", "127.0.0.1", "jsonplaceholder.typicode.com")
 
     private val okHttpClient = OkHttpClient.Builder()
         .addInterceptor(DatadogInterceptor(tracedHosts))
         .addNetworkInterceptor(TracingInterceptor(tracedHosts))
         .eventListenerFactory(DatadogEventListener.Factory())
         .build()
 
+    private val retrofitClient = Retrofit.Builder()
+        .baseUrl("https://jsonplaceholder.typicode.com")
```

Comment:
Although it can be a usefull tool, I'd rather have us use some public datadog owned "API" (eg: https://docs.datadoghq.com/api/), rather than relying on an unknown third party API.